### DISCUSSION TITLE
"Update: 00/12" What is this?

### DISCUSSION THREAD

**Discussion by Matsugae** [bangcorrupt/hacktribe#188] [Q&A] [answered]
I have this issue with my electribe, it suddenly appeared on reboot this message on the screen "Update: 00/12" Does anyone have any idea what this is?
I had not installed anything suddenly appeared when turning it on. There is no way to do anything, when restarting it reappears. I have tried to put the SD card with the original firmware file, but it does not even recognize it. Has it been bricked?

I made a short video showing the problem.
https://youtu.be/8ZcTSija2Bw

Any help is really appreciated.

**Comment by bangcorrupt** [comment, marked as answer]
When the CPU firmware boots, it queries the version of the MCU firmware.  If it does not receive the answer it is expecting, the CPU initiates an MCU firmware update.  If it is not getting past '00/12' it is likely the CPU can't communicate with the MCU.  The most likely culprit is the ribbon cable connecting the motherboard to the panel board, try re-seating or replacing it.  

If that doesn't work, you can check if the MCU on the panel board is running by attaching a logic analyser to the serial port, J3 shown bottom  centre in [this photo](https://1.bp.blogspot.com/-4ZRj5FRodss/VcfOcHV2VoI/AAAAAAAAAr4/P0Ma_-C6KSA/s1600/IMG_9300.JPG).

**Reply by Matsugae** [reply]
Many thanks for the answer, I'm going to open them and check the ribbon cable and MCU. Sounds like it's a manufacturing defect, because I bought it brandnew and had only used it a few times, but the warranty time already expired.
